sw/host/provisioning/ft/src/main.rs

```diff
@@ -199,11 +203,23 @@ fn main() -> Result<()> {
     // Once we are in a mission mode, we no longer need to provide a DFT strapping sequence on
     // every reset, as DFT is no longer enabled in mission modes.
     transport.ignore_dft_straps_on_reset()?;
+
+    let cert_endorsement_key_wrapper = match (
```

Comment:
This is probably dead code now. Can cleanup in a follow up.

Reply:
Well, we still need to figure out which of the two command line options to use, I could revert to the original 'if ...is_some()` but decided to leave it, was just  going to silence the default case, but this fell through the cracks.